pimd/pim_tib.c

```diff
@@ -106,6 +106,11 @@ bool tib_sg_gm_join(struct pim_instance *pim, pim_sgaddr sg,
 					  __func__, result);
 			return false;
 		}
+
+		if (!pim_addr_is_any(sg.src))
+			(*oilp)->oif_flags[pim_oif->mroute_vif_index] &=
```

Comment:
We should reuse UNSET_FLAG() here.

Reply:
Seems current pimd code doesn't use this style. Maybe somebody ( or i ) should create a new PR to reorganize it.